File: python/sum_of_even.py
def sum_of_even(n:list) -> int:
    if len(n) <= 2:
        raise ValueError("invalid array")
    sum = 0
    for i in n:
        if i % 2 == 0:
            sum += i

    return sum


def main():
    arr = [1,2,3,4,5,6]

    sum = sum_of_even(arr)
    print(f'The sum of the odd number is: {sum}')


main()

File: javascript/sumOfEven.js
function sumOfEven(arr){
    if(arr.length <= 0){
        return
    }

    let sum = 0

    for (let i=0; i <= arr.length; i++){
        if (i % 2 === 0){
            sum += i
        }
    }

    return sum
}

function main(){
    const arr = [1,2,3,4,5,6]

    sum = sumOfEven(arr)

    console.log(sum)
}

main()